tests/approved-cve-list.yaml

```diff
@@ -0,0 +1,33 @@
+
+generalwhitelist: #Approve CVE for any image
+  CVE-2019-12900: bzip2
```

Comment:
We must never whitelist without explanation. Each item whitelisted has to explain why it's whitelisted -- "Low severity" is sufficent for right now. For mediums we need a little more

Reply:
Removed CVEs below Medium severity and justified remaining 5.